adapters/repos/db/snapshot_integration_test.go

```diff
@@ -95,7 +95,7 @@ func TestSnapshot_IndexLevel(t *testing.T) {
 				//         - property__id/segment-123.wal
 				//         - property__id/segment-123.db
 				//     - 1 file from vector index commitlogger
-				assert.Len(t, snap.Files, 7)
+				assert.Len(t, snap.Files, 4) // .wal are excluded
```

Comment:
Would be good to update the above comment, removing the `*.wal` entries

Reply:
Good point!
Done